delivery_state/tests/test_delivery_state.py

```diff
@@ -21,15 +21,20 @@ def setUp(self):
             'product_id': product_shipping_cost.id,
             'fixed_price': 99.99,
         })
+        self.product = self.env["product.product"].create({
+            "name": "Test product",
+            "type": "product",
+        })
+        self.partner = self.env["res.partner"].create({
+            "name": "Mr. Odoo",
+        })
 
     def test_delivery_state(self):
-        product = self.env.ref('product.product_delivery_01')
-        partner = self.env.ref('base.res_partner_12')
         sale = self.env['sale.order'].create({
-            'partner_id': partner.id,
+            'partner_id': self.partner.id,
```

Comment:
just of curiosity...  any reason for this change?

Reply:
Well, we detected a problem in CIs of environments with default localizations. In this case, an EUR conversion from dollars is being made. So we try to avoid unexpected side effects creating the test data from scratch.  And I think it will be needed to create a pricelist as well, as it still makes the conversion...

Reply:
Thanks for the explanation. Good to know.